CMakeLists.txt

```diff
@@ -114,6 +115,16 @@ if(GIT_FOUND AND PYTHONINTERP_FOUND)
   if(NOT exit_code EQUAL "0")
     set(GIT_FIRMWARE_BTC_ONLY_VERSION_STRING "v0.0.0")
   endif()
+  execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} ./scripts/get_version firmware-bitboxbase --check-semver --check-gpg
```

Comment:
`firmware-bitboxbase-standard` (now that we have platform+edition; otherwise we could mistake bitboxbase to be an edition)

Here and in the rest of the diff.

Reply:
the tags follow the file names so far. do we really want them to follow the defines?